pkg/kyma/cmd/test/common.go

```diff
@@ -0,0 +1,93 @@
+package test
```

Comment:
I am not sure if it makes sense to have this file at all. It is usually discouraged to have things like common, ytils, etc...

Reviewing the contents, I see that all functions except `NewTestSuite` are used in exactly one place in one subcommand. So I would move them there.

For the ones that are used in more places, they can just be part of the main test command.

Reply:
I have refactored the code. There are 3 methods that used more than in 1 place, so I'd like to keep them here to avoid code duplication